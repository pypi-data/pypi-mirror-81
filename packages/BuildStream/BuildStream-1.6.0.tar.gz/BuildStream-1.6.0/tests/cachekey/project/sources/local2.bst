kind: import
sources:
- kind: local
  path: files/local
  directory: opt
