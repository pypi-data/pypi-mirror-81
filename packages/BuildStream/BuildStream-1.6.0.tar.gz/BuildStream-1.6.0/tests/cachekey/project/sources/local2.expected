33239f183d3cbaca957e9693637fca6cb91053d1c22f5c4dea577b50d0acc23d